src/StrawberryfieldHydroponicsService.php

```diff
@@ -181,64 +297,116 @@ public function run() {
 
         $pid = exec(
           sprintf("%s > /dev/null 2>&1 & echo $!", $cmd)
+          //sprintf("%s > /dev/null & echo $!", $cmd)
         );
         \Drupal::state()->set('hydroponics.queurunner_last_pid', $pid);
```

Comment:
Since State is being used a lot, we can better decency inject it.

Reply:
@DiegoPino I added State injection, please check I made all well, thanks

Reply:
Changed to StateInterface, please check when have some time if all ok as you commented.